check/tests/support/mod.rs

```diff
@@ -1,66 +1,71 @@
-extern crate env_logger;
+#![allow(private_in_public, dead_code)]
```

Comment:
Won't private_in_public be an error eventually? What is the reason for adding it?